apps/system/js/input_window_manager.js

```diff
@@ -226,17 +228,31 @@
           KeyboardManager._onKeyboardKilled(manifestURL);
         }
         break;
+
+      // We want to avoid the keyboard showing while opening
+      // an actionmenu/activity/attention. These are the event
+      // pairs:
+      // actionmenuopening - actionmenuopen
+      // activityrequesting - activityopening
```

Comment:
what happens if we cancel the activity from the picker? We won't get the `activityopening` event...